---
title: "WHI_GWASTools"
author: "anb"
date: "2/23/2021"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r}
library(GWASTools)
```

## African American

### Chromosome 1


```{r}

dosefile <- system.file("extdata", "imputation", "MaCH", "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/phg000592.v1.WHI_SHARE_aa.genotype-imputed-data.c1/SHAREchr1aa.dose.c1",
package="GWASdata")


markfile <- system.file("extdata", "imputation", "MaCH", "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/phg000592.v1.WHI_SHARE_aa.genotype-imputed-data.c1/SHAREchr1aa.info",
package="GWASdata")

posfile <- system.file("extdata", "imputation", "MaCH", "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/posfiles/aa/SHAREchr1aa_posfile.csv",
package="GWASdata")



```



```{r}
chr1 <- imputedDosageFile(input.files=c(dosefile, markfile, posfile), filename=gdsfile, chromosome=1, input.type="MaCH", input.dosage=TRUE, file.type="gds", snp.annot.filename=snpfile, scan.annot.filename=scanfile, genotypeDim="scan,snp",
                  scan.df=NULL,
                  snp.exclude=NULL,
                  snp.id.start=1,
                  block.size=5000, 
                  prob.threshold=0.9, verbose=TRUE)
```

    