---
title: "R Notebook"
output: html_notebook
---

```{r}
library(GWASTools)
```
# Consent Group 1

## African American

Load scan file into R
```{r}
scanfile_aa <- "/home/anbennett2/analysis/results/gwas_pancan/pheno/c1_SHARE_AA_pheno.csv"
```


### Chromosome 1
Load chromosome files into R:
```{r}
gdsfile <- "/home/anbennett2/analysis/results/gwas_pancan/pheno/gwastools/gdsfiles/chr1.gds"
snpfile <- "/home/anbennett2/chr1_snpannot"
dosefile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/phg000592.v1.WHI_SHARE_aa.genotype-imputed-data.c1/SHAREchr1aa.dose.c1"
markfile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/phg000592.v1.WHI_SHARE_aa.genotype-imputed-data.c1/SHAREchr1aa.info"
posfile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/posfiles/aa/SHAREchr1aa_posfile.csv"
```

```{r}
imputedDosageFile(input.files=c(dosefile, markfile, posfile), filename=gdsfile, chromosome=1, input.type="MaCH", input.dosage=TRUE, 
file.type="gds",snp.annot.filename=snpfile, scan.annot.filename=scanfile, scan.df=NULL, snp.exclude=NULL, prob.threshold=0.9, verbose=TRUE)
```


### Chromosome 21
Load chromosome files into R:
```{r}
gdsfile <- "/home/anbennett2/analysis/results/gwas_pancan/pheno/gwastools/gdsfiles/chr21.gds"
snpfile <- "/home/anbennett2/chr21_snpannot"
dosefile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/phg000592.v1.WHI_SHARE_aa.genotype-imputed-data.c1/SHAREchr21aa.dose.c1"
markfile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/phg000592.v1.WHI_SHARE_aa.genotype-imputed-data.c1/SHAREchr21aa.info"
posfile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/posfiles/aa/SHAREchr21aa_posfile.csv"
```


```{r}
imputedDosageFile(input.files=c(dosefile, markfile, posfile), filename=gdsfile, chromosome=21, input.type="MaCH", input.dosage=TRUE, 
file.type="gds",snp.annot.filename=snpfile, scan.annot.filename=scanfile, scan.df=NULL, snp.exclude=NULL, prob.threshold=0.9, verbose=TRUE)
```


