---
title: "WHI_GWASTools"
author: "anb"
date: "2/23/2021"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r}
library(GWASTools)
```

## African American

### Chromosome 1

Attach the files:
```{r}
gdsfile <- "/home/anbennett2/analysis/results/gwas_pancan/pheno/c1_SHARE_AA_pheno.csv"
snpfile <- "/home/anbennett2/chr1_snpannot"
dosefile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/phg000592.v1.WHI_SHARE_aa.genotype-imputed-data.c1/SHAREchr1aa.dose.c1"
markfile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/phg000592.v1.WHI_SHARE_aa.genotype-imputed-data.c1/SHAREchr1aa.info"
posfile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/posfiles/aa/SHAREchr1aa_posfile.csv"
```


```{r}
imputedDosageFile(input.files=c(dosefile, markfile, posfile), filename=gdsfile, chromosome=1, input.type="MaCH", input.dosage=TRUE, file.type="gds",snp.annot.filename=snpfile, scan.annot.filename=scanfile, scan.df=NULL, snp.exclude=NULL, snp.id.start=1, block.size=5000, prob.threshold=0.9, verbose=TRUE)
```


```{r}
imputedDosageFile(input.files=c(dosefile, markfile, posfile), filename=gdsfile, chromosome=1, input.type="MaCH", input.dosage=TRUE, 
file.type="gds",snp.annot.filename=snpfile, scan.annot.filename=scanfile, scan.df=NULL, snp.exclude=NULL, prob.threshold=0.9, verbose=TRUE)
```



### Chromosome 21

```{r}
scanfile <- "/home/anbennett2/analysis/results/gwas_pancan/pheno/c1_SHARE_AA_pheno.csv"
snpfile <- "/home/anbennett2/chr21_snpannot"
dosefile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/phg000592.v1.WHI_SHARE_aa.genotype-imputed-data.c1/SHAREchr21aa.dose.c1"
markfile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/phg000592.v1.WHI_SHARE_aa.genotype-imputed-data.c1/SHAREchr21aa.info"
posfile <- "/home/anbennett2/dbgap_data/WHI/consentgroup_1/phenogeno/geno/746/posfiles/aa/SHAREchr21aa_posfile.csv"


```

```{r}
imputedDosageFile(input.files=c(dosefile, markfile, posfile), filename=gdsfile, chromosome=21, input.type="MaCH", input.dosage=TRUE, 
file.type="gds",snp.annot.filename=snpfile, scan.annot.filename=scanfile, scan.df=NULL, snp.exclude=NULL, prob.threshold=0.9, verbose=TRUE)
```

```{r}
snpAnnot <- SnpAnnotationDataFrame(data.frame(snpID, chromosome, position, alleleA, alleleB, stringsAsFactors=FALSE))
?t.test()
```

```{r}
scanAnnot <- ScanAnnotationDataFrame(data.frame(scanID, SAMPLE_ID, SAMPLE_ORIGIN, dbGaP_Subject_ID, AGE, race, HISTORY_DIABETES, BLACK, PANCREATITIS, AGE_DIAGNOSED, DECEASED, AGE_DEATH, ENDWHIDY, AGE_END_FOLLOW, PAN_CAN, PAN_CAN_LOCATION, TUMOUR_BEHAVIOUR, DIAGNOSIS_STATUS, LATERALITY, TUMOUR_MORPHOLOGY, TUMOUR_GRADE, TUMOUR_SIZE, TUMOUR_STAGE   , MEAN_BMI, MEAN_WHR, PAN_CAN_MONTHS_SURVIVED, sex, stringsAsFactors=FALSE))
```


scanID, SAMPLE_ID, SAMPLE_ORIGIN, dbGaP_Subject_ID, AGE, race, HISTORY_DIABETES, BLACK, PANCREATITIS, AGE_DIAGNOSED, DECEASED, AGE_DEATH, ENDWHIDY, AGE_END_FOLLOW, PAN_CAN, PAN_CAN_LOCATION, TUMOUR_BEHAVIOUR, DIAGNOSIS_STATUS, LATERALITY, TUMOUR_MORPHOLOGY, TUMOUR_GRADE, TUMOUR_SIZE, TUMOUR_STAGE   , MEAN_BMI, MEAN_WHR, PAN_CAN_MONTHS_SURVIVED, sex



